EventCatalogAPI/Controllers/EventItemsController.cs

```diff
@@ -208,9 +209,46 @@ private List<EventItem> ChangeImageUrl(List<EventItem> items)
             return items;
         }
 
+        [HttpGet("[action]/Catagory/{catagoryId}/Type{typeId}")]
+        public async Task<IActionResult> Items(
+               [FromQuery] int? catagoryId,
+               [FromQuery] int? typeId,
+               [FromQuery] int pageIndx = 0,
```

Comment:
I think this should change from pageIndx to pageIndex. We might run into issues if we leave it

Reply:
ok  I will edit that